ui/components/app/alert-system/contexts/alertMetricsContext.tsx

```diff
@@ -1,17 +1,26 @@
 import React, { createContext, useContext, ReactNode, useMemo } from 'react';
-import { useConfirmationAlertMetrics } from '../../../../pages/confirmations/hooks/useConfirmationAlertMetrics';
 
 const AlertMetricsContext = createContext<{
   trackAlertActionClicked: (alertKey: string) => void;
```

Comment:
Minor, could we define a type for the track methods to avoid this duplication, such as `AlertMetricsCallbacks`?

Reply:
Done 👍